js/data/searchProviders.js

```diff
@@ -25,6 +25,7 @@ module.exports = { "providers" :
       "base" : "https://duckduckgo.com",
       "image" : "https://duckduckgo.com/favicon.ico",
       "search" : "https://duckduckgo.com/?q={searchTerms}&t=brave",
+      "privateSearch" : "https://duckduckgo.com/?q={searchTerms}&t=bravep",
```

Comment:
this opens up a privacy issue (duckduckgo can now distinguish whether someone is using private mode) :(

i think there are ways to avoid this leak while still satisfying DDG's requirement. will discuss more in other channels.